app/mailers/support_user_mailer.rb

```diff
@@ -10,4 +10,15 @@ def support_user_invitation(support_user)
 
     notify_email to: support_user.email, subject:, body:
   end
+
+  def support_user_removal_notification(support_user)
+    subject = t(".subject", service_name: t("#{support_user.service}.service_name"))
```

Comment:
Is the [service_name](https://github.com/DFE-Digital/itt-mentor-services/blob/main/app/helpers/application_helper.rb) helper available here?

Reply:
We've used `t("#{support_user.service}.service_name")` and `t("#{user.service}.service_name")` in all other mailers. I feel like it's a more guaranteed way of ensuring that the email relates to the service associated with the user, rather than the service it is being sent by.